Neuron/Models/Realm/TokenModel.swift

```diff
@@ -16,13 +16,11 @@ class TokenModel: Object, Decodable {
     @objc dynamic var address = ""
     @objc dynamic var decimals = 18
     @objc dynamic var symbol = ""
-    @objc dynamic var chainName: String? = ""
-    @objc dynamic var chainId = ""
-    @objc dynamic var chainHosts = "" // manifest.json chainSet.values.first
     @objc dynamic var identifier = UUID().uuidString
 
     // defaults false, eth and RPC "getMateData" is true.
-    @objc dynamic var isNativeToken = false // TODO: AppChain ERC20 should not be marked as native token.
+    @objc dynamic var isNativeToken = false
+    @objc dynamic var chain: ChainModel?
 
     @objc dynamic var balanceText = "0"
```

Comment:
`balanceText` is better to be `balance`.

Reply:
There is a `balance` use `BigUInt` type under `balanceText`. Realm Object can't be `BigUInt` type.